modules/cjson/CMakeLists.txt

```
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+set(CJSON_MODULE_DIR ${ZEPHYR_BASE}/../modules/lib/cjson)
+
+zephyr_library_named(cjson)
+zephyr_include_directories(
+	${CJSON_MODULE_DIR}
```

Just use `ZEPHYR_CJSON_MODULE_DIR` here:
```suggestion
	${ZEPHYR_CJSON_MODULE_DIR}
```